warehouse/query-core/src/test/resources/datawave/query/QueryLogicFactory.xml

```diff
@@ -236,6 +236,7 @@
         <property name="numRangesToBuffer" value="15" />
         <property name="rangeBufferTimeoutMillis" value="15000" />
         <property name="rangeBufferPollMillis" value="100" />
+        <property name="maxLinesToPrint" value="100"/>
```

Comment:
default to unlimited?

Reply:
Sure, I don't mind doing that for this QLF.xml. I think I'll leave the configuration in the EventQueryLogicFactory so there's at least some unit tests using this code path.